### ISSUE TITLE
[rocminfo] rocm_agent_enumerator fails if topology dir does not exist

### ISSUE THREAD

**Issue by acxz** [rocm-arch/rocm-arch#760]
```
Traceback (most recent call last):
  File "/opt/rocm/bin/rocm_agent_enumerator", line 257, in <module>
    main()
  File "/opt/rocm/bin/rocm_agent_enumerator", line 241, in main
    target_list = readFromKFD()
  File "/opt/rocm/bin/rocm_agent_enumerator", line 193, in readFromKFD
    for node in sorted(os.listdir(topology_dir)):
FileNotFoundError: [Errno 2] No such file or directory: '/sys/class/kfd/kfd/topology/nodes/'
```
Upstream issue: https://github.com/RadeonOpenCompute/rocminfo/issues/56
Upstream PR: https://github.com/RadeonOpenCompute/rocminfo/pull/53

**Comment by tpkessler** [comment]
Patched with #761

**Comment by acxz** [comment]
Closing this issue as PKGBUILD moved over to [community].
